canonical/cloud-init
# Fix cli debugging

## Additional Context
Issue reported by @a-dubs. Please test it if you don't mind. Rebasing your test branch on this one should work, I think.

## Test Steps
Tested using this minimal test case:
```python
import logging
from cloudinit import log

log.configure_root_logger()
log.add_log_handler_cli(logging.INFO)
LOG = logging.getLogger()
LOG.warning("hi")
```
Without this branch, no log is emitted. With this branch, the log appears on stderr,

## Merge type
